src/app/static/src/js/components/runReport/parameterList.vue

```diff
@@ -14,6 +14,7 @@
             </tr>
             </tbody>
         </table>
+        <div class="text-danger small">{{error}} </div>
```

Comment:
Looks like a superfluous space here?

Reply:
Yes, letting `parameter-list` manage validation and setting the error occurred to me - cleaner separation as you say, and reducing complexity of `run-report`. The reason I didn't recommend it is that it would need, I think, to emit an event when `isParamsValid` changed, so `run-report`could respond by disabling the button, which would be a bit fiddly but a completely valid approach. Actually I guess you could manage emitting that event with a watch on the computed prop in `parameter-list` so that;s probably simpler than I thought!